build/ip-payment@0.7.1.html

```diff
@@ -107,7 +107,11 @@ <h1 class="title">
                     <div class="dropdown-menu" id="dropdown-menu" role="menu">
                         <div class="dropdown-content">
                         
-                            <a href="ip-payment@0.7.0.html" class="dropdown-item is-active">
+                            <a href="ip-payment@0.7.1.html" class="dropdown-item is-active">
+                                ip-payment@0.7.1
+                            </a>
+                        
+                            <a href="ip-payment@0.7.0.html" class="dropdown-item ">
```

Comment:
Should this item be removed? 0.7.0 doesn't exist any more, right?

Reply:
Good catch! Just pushed a fix.